scripts/qa_crowbarsetup.sh

```diff
@@ -4184,7 +4184,8 @@ function oncontroller_testsetup
 
     wait_for 40 5 "timeout -k 20 10 ssh -o UserKnownHostsFile=/dev/null $ssh_target true" "SSH key to be copied to VM"
 
-    if ! ssh $ssh_target curl $clouddatadns/test ; then
+    [[ $http_proxy ]] && http_proxy_env="http_proxy=$http_proxy" || http_proxy_env=""
+    if ! ssh $ssh_target $http_proxy_env curl -s --fail http://suse.com ; then
```

Comment:
The old url was nice because it just contained the text content 'OK'. Even with -s --fail this outputs an HTML 301 page that at a casual glance might give the impression something had gone wrong. Could consider adding a >/dev/null to make it nicer.